src/jquery.ime.js

```diff
@@ -160,12 +160,12 @@
 		 * @param {string} input
 		 * @param {string} context
 		 * @param {boolean} altGr whether altGr key is pressed or not
-		 * @return {Object} Transliteration object
-		 * @return {boolean} return.noop Whether to consider input processed or passed through.
-		 * @return {string} return.output The transliterated input or input unmodified.
+		 * @return {Object} Transliteration object, with the following fields:
```

Comment:
eslint warned about multiple return statements in JSDoc, so I changed it into one.